internal/common/config/clientconfig.go

```diff
@@ -8,6 +8,33 @@ type ClientConfig struct {
 	MinimumResourceThreshold    shared.Resources
 	MaxCriticalConsecutiveTurns uint
 	DisasterConfig              ClientDisasterConfig
+	IIGOClientConfig            ClientIIGOConfig
+}
+
+// ClientIIGOConfig contains iigo config fields that is visible to clients
+type ClientIIGOConfig struct {
+	// Executive branch
+	GetRuleForSpeakerActionCost        SelectivelyVisibleResources
```

Comment:
just use shared.Resource or int no need to use the selectively visible type

Reply:
Ah right ok, that saves a lot of lines.